CONTRIBUTING.md

```diff
@@ -264,6 +273,55 @@ Finally, when a point release is ready to be made:
 Note, although we aim to support only a single release at a time, the process stated above could be
 used for multiple previous versions.
 
+## Stable Release Updates (SRU)
+
+Once a Cosmos-SDK release has been completed and published, updates for it are released under certain circumstances
+and must follow the *Stable Release Update* (or SRU) procedure.
+
+### Rationale
```

Comment:
@marbar3778 @alexanderbez here is the rationale. Thoughts? Does that make sense to you?

Reply:
thank you for the rationale. I do agree but still dont understand the naming change. Every release is a stable release or should be. We can keep the rationale because it will help but there isnt a need for a rename IMO.

Reply:
What do you mean by every release? A `major.minor` is a major stable release (e.g. 0.38), backports/bugfixes/updates go into _point_ releases. Am I wrong?